zlib/conn.go

```diff
@@ -618,3 +622,12 @@ func (c *Conn) SSHHandshake() error {
 	c.grabData.SSH = handshakeLog
 	return err
 }
+
+func (c *Conn) ISCSIScan(config *iscsi.ISCSIConfig) error {
+	data, err := iscsi.Scan(c.getUnderlyingConn(), config)
+	if err != nil {
```

Comment:
Are you sure this doesn't throw away data? What if we timeout halfway through, we still want to record whatever we were able to get.